docs/data/pages.ts

```diff
@@ -64,7 +64,22 @@ const pages: MuiPage[] = [
       },
       { pathname: '/x/react-data-grid/editing' },
       { pathname: '/x/react-data-grid/sorting' },
-      { pathname: '/x/react-data-grid/filtering' },
+      {
+        pathname: '/x/react-data-grid/filtering',
+        title: 'Filtering',
+        children: [
+          { pathname: '/x/react-data-grid/filtering/overview' },
+          { pathname: '/x/react-data-grid/filtering/customization' },
```

Comment:
Would it be clearer?

```suggestion
          { pathname: '/x/react-data-grid/filtering/customization', title: 'Filter customization' },
```

Reply:
IMO it'd be obvious as it's under `Filtering` section anyways.

Reply:
Yeah, OK, it makes sense